ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -251,3 +251,16 @@ def unique(input, sorted=True, return_inverse=False, return_counts=False, dim=No
         values.append(results.counts)
 
     return Results(*values)
+
+
+@to_ivy_arrays_and_back
+def mode(input, dim=-1, keepdim=False, *, out=None):
+    sorted_data = sorted(input.items(), key=lambda x: x[1], reverse=True)

```

Comment:
There is no attribute `items`. 
You can use ivy functions in the frontends. Using `ivy.sort` directly on `input` would make things much easier.